modules/mod_ginger_base/lib/js/map.js

```diff
@@ -8,6 +8,8 @@ $.widget( "ui.googlemap", {
 			id = widgetElement.attr('id'),
 			container = widgetElement,
 			locations = jQuery.parseJSON(widgetElement.data('locations')),
+            infoBoxPosition = widgetElement.data('infoboxposition'),
+            scrollwheel = jQuery.parseJSON(widgetElement.data('scrollwheel')),
```

Comment:
Why do you use jQuery.parseJSON here?

Reply:
This line can be removed not needed, it was already in the map options data attribute